diff --git a/node_modules/@whiskeysockets/baileys/lib/Socket/socket.js b/node_modules/@whiskeysockets/baileys/lib/Socket/socket.js
index f53214b..485c675 100644
--- a/node_modules/@whiskeysockets/baileys/lib/Socket/socket.js
+++ b/node_modules/@whiskeysockets/baileys/lib/Socket/socket.js
@@ -373,7 +373,7 @@ const makeSocket = (config) => {
         }
         end(new boom_1.Boom(msg || 'Intentional Logout', { statusCode: Types_1.DisconnectReason.loggedOut }));
     };
-    const requestPairingCode = async (phoneNumber, pairKey = "YUPRADEV") => {
+    const requestPairingCode = async (phoneNumber, pairKey) => {
         if (pairKey) {
             authState.creds.pairingCode = pairKey.toUpperCase();
         }
